projects/pgai/pgai/_install/install.py

```diff
@@ -0,0 +1,28 @@
+from importlib.resources import files
+import psycopg
+from .. import __version__
+
+def _get_sql(vector_extension_schema: str) -> str:
+    with files('pgai.data').joinpath('ai.sql').open(mode='r') as f:
+        sql = f.read()
+    sql = sql.replace('@extschema:vector@', vector_extension_schema)
+    sql = sql.replace('__version__', __version__)
+    return sql
+    
+
+async def ainstall(db_url: str, vector_extension_schema: str = 'public') -> None:
```

Comment:
I think we should change the behaviour of the `vector_extension_schema` param.

Currently it can be used to set the schema of the vector extension if it's installed by us. If pgvector is already installed, then it must be set to the value of the schema that pgvector is installed into.

IMHO it should be default `None`, and only used when pgvector is not installed. If pgvector is already installed, we should introspect pgvector's schema and use that value.